specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -469,19 +494,19 @@ def verify_cell_proof_batch(row_commitments_bytes: Sequence[Bytes48],
 
     Public method.
     """
-    assert len(cells_bytes) == len(proofs_bytes) == len(row_indices) == len(column_indices)
+    assert len(cells) == len(proofs_bytes) == len(row_indices) == len(column_indices)
 
     # Get commitments via row IDs
     commitments_bytes = [row_commitments_bytes[row_index] for row_index in row_indices]
 
     # Get objects from bytes
     commitments = [bytes_to_kzg_commitment(commitment_bytes) for commitment_bytes in commitments_bytes]
-    cells = [bytes_to_cell(cell_bytes) for cell_bytes in cells_bytes]
+    cosets_evals = [cell_to_coset_evals(cell) for cell in cells]
```

Comment:
`coset_evals` is a list of `CosetEvals` here. Might be a bit confusing because the struct name is already in plural...